docs/runtime_suite/form-service-backend/40_submit_urls.md

```diff
@@ -59,7 +59,7 @@ Each property of the `data` object is related to a component of the form.
 ## GET /export route
 The `GET /export` route returns the submitted data of the forms matching the query parameters and must be compatible with the API of the endpoint of the [CRUD Service](../crud-service/overview_and_usage). Specifically, this endpoint must support filtering data using the `_q` query parameter or any form data field.
 
-This endpoint must return a list of JSON objects in [newline delimited JSON](https://en.wikipedia.org/wiki/JSON_streaming#Newline-delimited_JSON) format.
+This endpoint must return a list of JSON objects in [newline delimited JSON](http://ndjson.org/) format.
```

Comment:
This domain has been squatted, please remain on wikipedia or else link the github ndjson page

Reply:
We restored the Wikipedia URL in the doc, I will review it tomorrow.